cloud/cluster/fetch_cron_test.go

```diff
@@ -0,0 +1,81 @@
+package cluster_test
+
+import (
+	"testing"
+	"github.com/scootdev/scoot/cloud/cluster"
+	"time"
+	"reflect"
+	"sync"
+	"sort"
+	// "fmt"
+)
+
+func TestFetchCron(t *testing.T) {
+	h := makeCronHelper(t)
+	h.assertFetch(t, h.setupTest("host1:1234"))
+	h.assertFetch(t, h.setupTest("host1:1234", "host2:8888"))
+	h.assertFetch(t, h.setupTest("host1:1234", "host1:1234"))
+	h.assertFetch(t, h.setupTest(""))
+}
+
+func (h *cronHelper) setupTest(nodeNames ...string) []cluster.Node {
+	nodes := nodes(nodeNames)
+	h.f.setResult(nodes)
+	time.Sleep(h.time * 2)
```

Comment:
without a sleep the fetched state didn't have enough time to propagate to the state and back to the subscribers